packages/blade/src/hooks/__tests__/useBladeInnerRef.web.test.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import type { BladeElementRef } from '../useBladeInnerRef';
+import { useBladeInnerRef } from '../useBladeInnerRef';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+
+describe('useBladeInnerRef', () => {
+  it('should only expose focus and scrollIntoView', () => {
+    let refValue = null;
```

Comment:
why not 
```suggestion
    let ref = React.useRef(null);
```

Reply:
If you notice, it's a temporary variable outside the React component. I'm using this variable to hold the value of the ref so that I can assert it later. `expect(refValue).toHaveProperty('focus');`